docs/faq.md

```diff
@@ -311,4 +311,9 @@ let populated = GREYCondition(name: "Wait for UICollectionView to populate", blo
 }).waitWithTimeout(20.0)
 
 GREYAssertTrue(populated, reason: "Failed to populate UICollectionView in 20 seconds")
-```
\ No newline at end of file
+```
+
+**How do I match AX=N elements?**
+
+Elements with accessibility disabled (`AX=N`) can still be matched by `grey_accessibilityID`. The other accessibility matchers (`grey_accessibilityLabel`, `grey_accessibilityValue`, `grey_accessibilityTrait`, `grey_accessibilityHint`) won't work. If the `AX=N` element can't be matched by id, then
```

Comment:
AX=N is a specific thing in the UI hierarchy. Maybe add that it's present in the UI hierarchy only.